### PR TITLE
*: update etcd

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
- Fix the prevote issue about cannot elect a new leader
- Add the `zap` logger builder 
### What is changed and how it works?
update etcd to the latest version

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: pkg/logutil/log.go [hunk 1]
```diff
@@ -23,10 +23,10 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/coreos/etcd/raft"
 	"github.com/coreos/pkg/capnslog"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
+	"go.etcd.io/etcd/raft"
```
Comment (tiancaiamao) [inline pkg/logutil/log.go]: Why "github.com/coreos/etcd" is changed to "go.etcd.io/etcd" ? @nolouch

Comment (nolouch) [reply pkg/logutil/log.go]: `github.com/coreos/etcd` has renamed to use package name `go.etcd.io/etcd`

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**